libraries/Microsoft.Bot.Builder.Ai/LocaleConverterMiddleware.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.Bot.Builder.Middleware;
+using Microsoft.Bot.Schema;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.Ai
+{
+    /// <summary>
+    /// Middleware to convert messages between different locales specified
+    /// </summary>
+    public class LocaleConverterMiddleware : IReceiveActivity
+    {
+        private LocaleConverter localeConverter;
+        private string fromLocale;
+        private string toLocale;
+
+        public LocaleConverterMiddleware(string fromLocale, string toLocale)
+        {
+            localeConverter = new LocaleConverter();
```

Comment:
>LocaleConverter [](start = 34, length = 15)

it seems like there should be a single shared instance of this class?

instantiating objects in the constructor is a DI anti-pattern - would you prefer to take a LocaleConverter interface?

Reply:
I updated the latest commit to have an interface for the locale converter for  to be used with Unity DI